### DISCUSSION TITLE
How to prevent non-native preview from blocking drop targets?

### DISCUSSION THREAD

**Discussion by tdcook** [atlassian/pragmatic-drag-and-drop#155] [Q&A]
I have followed the advice in the [docs for non-native drag previews](https://atlassian.design/components/pragmatic-drag-and-drop/core-package/adapters/element/drag-previews#non-native-custom-drag-previews) to render the preview in a portal and position it under the cursor, but when doing so it appears to block the drop targets. 

In my codesandbox below I've built an example of this. The squares should turn yellow when they are active drop targets, but when using the non-native preview, the drop targets are never activated.

https://codesandbox.io/p/sandbox/pdnd-portal-test-forked-j9tzrx

**Comment by hungify** [comment]
Your code doesn't follow the correct technique.

Fortunately, I have completed this implementation. Check it out [here](https://github.com/atlassian/pragmatic-drag-and-drop/issues/162#issuecomment-2543457720).

**Reply by tdcook** [reply]
It would be helpful if you explained what the correct technique is, but either way I decided to use [dnd-kit](https://github.com/clauderic/dnd-kit) instead, so I am no longer seeking an answer.
